packages/api-docs/src/markdown/generateMarkdown.js

```diff
@@ -1,12 +1,14 @@
-import generateDescription from './generateDescription';
 import generateTitle from './generateTitle';
+import generateImportText from './generateImportText';
+import generateDescription from './generateDescription';
 import generateProps from './generateProps';
 
-export default function generateMarkdown(name, reactAPI, imagePath) {
-  const markdownString = `${generateTitle(name)}\n${generateDescription(
-    reactAPI.description,
-    imagePath,
-  )}\n${generateProps(reactAPI.props)}\n`;
-
+export default function generateMarkdown(name, folderName, reactAPI) {
```

Comment:
Can possibly be rewrote as an arrow function because it is in ES6.

```javascript
const generateMarkdown = (name, folderName, reactAPI) => {
  const markdownString = `${generateTitle(name)}
  ${generateImportText(name, folderName)}
  <!-- STORY -->
  ${generateDescription(reactAPI.description)}
  ${generateProps(reactAPI.props)}
  `;
  return markdownString;
};

export default generateMarkdown;
```

Reply:
Good spot, it may have been left as a named function for easier debugging, but not sure. Lets ask Alasdair about this when he is back!